lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -373,6 +656,30 @@ impl Client {
         }
         Err(RdpError::TryError("no active device ids".to_string()))
     }
+
+    fn generate_file_id(&mut self) -> u32 {
+        self.next_file_id = self.next_file_id.wrapping_add(1);
+        self.next_file_id
+    }
+}
+
+#[allow(dead_code)]
+struct FileCacheObject {
```

Comment:
Please add doc a comment here. What's a file cache object for? How long is it cached?

Reply:
See 

https://github.com/gravitational/teleport/pull/12665/commits/4fbb4d7f34afd8a1a7b7e6f0472f762402350570#diff-983433889acb8e2c478d0a04b26bb477bd81271f29515f1ee6c56fa881e2880aR669-R685